hoomd/md/nlist.py

```diff
@@ -119,10 +124,12 @@ class Cell(_NList):
 
     Args:
```

Comment:
Document all of these arguments, except for `deterministic` under an `Attributes:` section of the base class `_Nlist` (you can copy and paste). `deterministic` should be documented under an `Attributes:` section of `Cell` at the end of the class docstring.